src/Stack.php

```diff
@@ -11,7 +11,8 @@
  * @package Ds
  *
  * @template TValue
- * @implements Sequence<TValue>
+ * @implements Sequence<int, TValue>
```

Comment:
I think you meant to use `Collection` here?
```suggestion
 * @implements Collection<int, TValue>
```